pkg/ingester/ingester_v2.go

```diff
@@ -782,3 +792,7 @@ func (i *Ingester) shipBlocks() {
 	// Wait until all workers completed.
 	wg.Wait()
 }
+
+func (db *userTSDB) currentNumSeries() uint64 {
+	return db.Head().NumSeries()
+}
```

Comment:
No need for this new method. It is fine to call `db.Head().NumSeries()` wherever needed.

Reply:
ok. i'll fix it. thanks!